### DISCUSSION TITLE
Checks on draft PRs do not update after the first run

### DISCUSSION THREAD

**Discussion by OisinMoran** [community/community#5030] [Projects and Issues]
When changes are pushed to a draft PR on a branch that has checks enabled, even though the checks are run again (in CircleCI) and have different results there, these changes do not show up on the draft PR. Also—unsurprisingly—clicking through "Details" only brings you to the out-of-date run of the checks.

For example, this is what Github shows right now (clicking though takes me to a run from 20h ago in this case):
![image](https://user-images.githubusercontent.com/8939851/129032923-d9e83b8a-f0de-4484-85b6-27fbe3c7bae6.png)

And this is what CircleCI shows (from 1h ago) when you manually navigate to the correct run.
![image](https://user-images.githubusercontent.com/8939851/129033458-8d0a215c-3b12-4c28-887c-68a70b80e5e8.png)

This could be an issue with Github or CircleCI (or both) but my estimate is around 80% likely that it is just a Github issue (mainly because I don't think CircleCI knows anything about whether a PR is a draft or not, and also I have previously experiences cache type of issues with Github).

Happy to provide any extra context, try out fixes, or answer any questions about this—has been bothering me for at least a year.

**Comment by Brunomachadob** [comment]
I can confirm this also happens in my company, but using Jenkins.
